homework/burgers/public/javascripts/inventory.js

```diff
@@ -0,0 +1,113 @@
+var $addnewf = $("#addnewitemform");
+var $addbtn = $("#addnewbtn");
+var $npricein = $("#npricein");
+var $nnamein = $("#nnamein");
+var $nquanin = $("#nquanin");
+var counter = $(".badge");
+var $updatebuttons = $("button.price, button.name, button.quantity");
+var $outofstockbuttons = $("button.textbutton");
+
+var outofstockcolor = function () {
+  // Colors quantity inputs of out of stock items light red.
+  $("input.quantity[placeholder=0]").css('background-color', '#FFADAD');
```

Comment:
We didn't teach this, but since you tried it anyway, you should use a class for this instead of changing specific css properties.